src/utils.ts

```diff
@@ -361,3 +362,18 @@ export const nFormatter = (num: number, digits: number): string => {
     (num / si[index].value).toFixed(digits).replace(rx, "$1") + si[index].symbol
   );
 };
+
+export const getVersion = () => {
+  const version = document.querySelector('meta[name="version"]');
+  return version ? (version as any).content : DEFAULT_VERSION;
+};
+
+export const getLatestVersion = async () => {
```

Comment:
You're never using this. Let's remove it.

Reply:
I believe it's a wip for https://github.com/excalidraw/excalidraw/pull/2731#issuecomment-755281227